src/User/phpstan.neon

```diff
@@ -0,0 +1,13 @@
+parameters:
+
+    autoload_files:
+        - vendor/bin/.phpunit/phpunit-6.3/vendor/autoload.php
+
+    ignoreErrors:
+        # Ignored so that phpstan/phpstan-phpunit doesn't have to be installed
+        - '#PHPUnit_Framework_MockObject_MockObject given#'
+        # PHPDoc bug in SF
+        - '#Strict comparison using === between null and string will always evaluate to false#'
```

Comment:
can it be more specific to `SF\Console::ask()`? not sure what the previous message was exactly ^^